### PR TITLE
feat: Gallery UI improvements

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Project automation
- Other... Please describe:

## What is the current behavior?

- Top level menu items looked plain with no iconography
- Navigation menu items foreground color was hard to read, especially when projected
- The introduction texts font size was too large and looked inconsistent, especially on smaller screens
- **Features** section contained some UI and non-UI features, and even some controls

## What is the new behavior?

- Adjusted foreground color for menu
- Adjusted font size for overview header
- Adjusted categorization
- Added icons for top-level items

image

Screenshot_20221006-142157

Screenshot_20221006-142124

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [ ] Tested on Android.
- [ ] Tested on UWP.
- [ ] Tested in both **Light** and **Dark** themes.
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by amelield** [comment]
I think it looks really good!

File: Uno.Gallery/Uno.Gallery.Shared/Entities/SampleCategory.cs [hunk 1]
```diff
@@ -12,9 +9,19 @@ public enum SampleCategory
...
+		[SampleCategoryInfo("\uE81E", "UI components")]
+		UIComponents,
+        
+        [SampleCategoryInfo("\uE8AE", "UI features")]
+        UIFeatures,
+
+        [SampleCategoryInfo("\uE950", "Non-UI features")]
+        NonUIFeatures,
+        
+        [SampleCategoryInfo("\uF0B4", "Toolkit")]
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.Shared/Entities/SampleCategory.cs:24]: Fix mixed whitespace before @Xiaoy312 gets mad at you 😆

File: Uno.Gallery/Uno.Gallery.UWP/App.xaml.cs [hunk 2]
```diff
@@ -251,29 +252,32 @@ private void AddNavigationItems(MUXC.NavigationView nv)
...
+                    {
+                        Content = sample.Title,
+                        Icon = !string.IsNullOrEmpty(sample.Glyph) ? new FontIcon() { Glyph = sample.Glyph } : null,
+                        DataContext = sample,
+                        Style = (Style)Resources[$"T{tier}NavigationViewItemStyle"]
+                    }.Apply(NavViewItemVisualStateFix);
+                    AutomationProperties.SetAutomationId(item, "Section_" + item.Content);
+
+                    (parentItem?.MenuItems ?? nv.MenuItems).Add(item);
+                }
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/App.xaml.cs:280]: mixed whitespace

File: Uno.Gallery/Uno.Gallery.UWP/Views/Styles/NavigationView.xaml [hunk 3]
```diff
@@ -2606,15 +2606,16 @@
 	<Style x:Key="T1NavigationViewItemStyle"
 		   BasedOn="{StaticResource DefaultNavigationViewItemStyle}"
 		   TargetType="muxc:NavigationViewItem">
-		<Setter Property="Foreground" Value="{StaticResource OnSurfaceLowBrush}" />
+        <Setter Property="Foreground" Value="{ThemeResource TextFillColorPrimaryBrush}" />
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/Styles/NavigationView.xaml:2609]: mixed whitespace

File: Uno.Gallery/Uno.Gallery.UWP/Views/Styles/NavigationView.xaml [hunk 4]
```diff
@@ -2606,15 +2606,16 @@
...
 	</Style>
 	
 	<Style x:Key="T2NavigationViewItemStyle"
 		   BasedOn="{StaticResource DefaultNavigationViewItemStyle}"
 		   TargetType="muxc:NavigationViewItem">
-		<Setter Property="FontSize" Value="14" />
-	</Style>
+        <Setter Property="Foreground" Value="{ThemeResource TextFillColorSecondaryBrush}" />
+        <Setter Property="FontSize" Value="14" />
+    </Style>
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/Styles/NavigationView.xaml:2618]: mixed whitespace

File: Uno.Gallery/Uno.Gallery.UWP/Views/GeneralPages/OverviewPageHeader.xaml [hunk 5]
```diff
@@ -7,6 +7,22 @@
 			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
```
Comment (kazo0) [inline Uno.Gallery/Uno.Gallery.UWP/Views/GeneralPages/OverviewPageHeader.xaml:7]: mixed whitespace

**Review by kazo0 — APPROVED** [review]

**Pull request merged**